Customise readme page (#57)
AY2324S2-CS2103-F08-2/tp: README.md

![Ui](docs/images/Ui.png)

> * This is our **CogniCare appliction**. pu<br>

Reviewer: ```suggestion
* This is our **CogniCare application**. pu<br>
```

  Example usages:

>   * allows BS to create contacts in the CogniCare application

Reviewer: ```suggestion
  * allows counsellors to create contacts in the CogniCare application
```

>   * allows BS to track his student's mental health progress

Reviewer: ```suggestion
  * allows counsellors to track his student's mental health progress
```

* The project is an ongoing software project for a desktop application (called _CongiCare_) which is used for managing student contact details.
  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base.
  * It comes with a **reasonable level of user and developer documentation**.

> * We hope that the application will help all SoC students to achieve better mental health, and BS to spend more time to help students instead of wasting time on administrative work
> * by helping BS keep track of student information.

Reviewer: ```suggestion
* We hope that the application will help all SoC students to achieve better mental health, and counsellors to spend more time to help students instead of wasting time on administrative work by helping them keep track of student information.
```
